app/assets/stylesheets/devices.scss

```diff
@@ -88,7 +88,7 @@ subhead {
 }
 
 .ui-head.wacubo {
- background-color: rgba(38,50,71,0.9); 
+ background-color: rgba(13,101,165,0.9); 
```

Comment:
Rather than use a magic value here, let's set a SASS variable:

- [app/assets/stylesheets/base/_variables.scss](https://github.com/chapmanu/signage/blob/0c62f090b92808cd3d3200b78988cad6ce188f4d/app/assets/stylesheets/base/_variables.scss)


Say,

    $wacubo-blue: rgba(13,101,165,0.9);